### PR TITLE
Support Purifier Humidify+Cool Formaldehyde

### PR DESCRIPTION
**Pull request by vwt12eh8**
The following support has been added.
https://www.dyson.com/air-treatment/air-purifier-humidifiers/purifier-humidify-cool-formaldehyde-ph04

Device type is 358E.
It extends from the already implemented "device type 358".
Supports the acquisition of formaldehyde sensor values.

### CODE REVIEW

File: libdyson/__init__.py [hunk 1]
```diff
@@ -64,4 +65,6 @@ def get_device(serial: str, credential: str, device_type: str) -> Optional[Dyson
         return DysonPureHotCool(serial, credential, device_type)
     if device_type == DEVICE_TYPE_PURE_HUMIDIFY_COOL:
         return DysonPureHumidifyCool(serial, credential, device_type)
+    if device_type == DEVICE_TYPE_PURIFIER_HUMIDIFY_COOL_FORMALDEHYDE:
```
Comment (Kakise) [inline libdyson/__init__.py:68]: It would be great to add tests for this part

**Review by Kakise — CHANGES_REQUESTED** [review]

File: libdyson/__init__.py (hunk 1 above)
Comment (vwt12eh8) [reply libdyson/__init__.py:68]: I'm sorry. Fixed.

**Review by Kakise — APPROVED** [review]

**Comment by Kakise** [comment]
LGTM ! thanks for your contribution ! :) It will be available around this weekend alongside 0.16.1 release

**Comment by austinbeam** [comment]
This device type also applies to the PH03 Model, one of which I received as a warranty replacement for my PH01. Is there any update @Kakise on when this commit will be included in a release? I cannot currently connect to my device with ha-dyson because the device type is not supported in libdyson.

Thanks!

**Pull request merged**